index.html

```diff
@@ -10,24 +10,59 @@
   <link rel="stylesheet" href="./styles/styles.css">
 </head>
 <body>
+<section class="topsection">
+  <div class="container topsection__container">
+    <div class="topsection__header">
+      <div class="topsection__logo"></div>
+      <div class="topsection__header__items">
+        <ul class="topsection__header__links">
+          <li class="topsection__header__link active">Stays</li>
+          <li class="topsection__header__link">Attractions</li>
+        </ul>
+        <div class="topsection__header__buttons">
+          <div class="topsection__button__night"></div>
+          <div class="topsection__button__account"></div>
+        </div>
+      </div>
+    </div>
+    <h1 class="title topsection__title">Discover stays<br> to live, work or just relax</h1>
+    <div class="form">
+      <form class="topsection__form" action="./" method="get">
+        <label class="input__label topsection__input__label" for="city">Your destination or hotel name</label>
+        <label class="input__label topsection__input__label" for="date">Check-in &ndash; Check-out</label>
+        <label class="input__label topsection__input__label" for="guests"></label>
```

Comment:
Label without content doesn't make sense for me.
Also, as the labels are separated from inputs - they appears in different way with inputs on screens with different width.